pkg/sentry/devices/nvproxy/version.go

```diff
@@ -269,6 +269,7 @@ func Init() {
 					nvgpu.NV2080_CTRL_CMD_RC_RELEASE_WATCHDOG_REQUESTS:                     rmControlSimple,
 					nvgpu.NV2080_CTRL_CMD_RC_SOFT_DISABLE_WATCHDOG:                         rmControlSimple,
 					nvgpu.NV2080_CTRL_CMD_TIMER_GET_GPU_CPU_TIME_CORRELATION_INFO:          rmControlSimple,
+					nvgpu.NV2080_CTRL_CMD_PERF_GET_CURRENT_PSTATE:                          rmControlSimple,
```

Comment:
NV2080_CTRL_CMD_PERF_GET_CURRENT_PSTATE was only added into the GPU driver at version 550.40.07. https://github.com/NVIDIA/open-gpu-kernel-modules/commit/91676d6628d3fa3a4d23ff7d6a4ee045d1a29654#diff-9b38fdaeb177fb8c24295707900714055653d053639408439af524f11f1fe6d2

So you'd have to set this entry below in `v550_40_07()`'s definition.

Reply:
Should be fixed now.